### DISCUSSION TITLE
dhcp range

### DISCUSSION THREAD

**Discussion by albertodbs** [Ysurac/openmptcprouter#1773] [Q&A] [answered]
Hello,
I have a problem with a tp-link access point that does not respect the dhcp range set on openmptcprouter, do you have any suggestions

**Comment by Ysurac** [comment]
What you mean ? Client get IP not on OpenMPTCProuter DHCP defined range ? In this case tp-link DHCP server is disabled or set as relay ?

**Comment by albertodbs** [comment, marked as answer]
I found the problem, another access point with active dhcp was installed and I was not notified of the installation. once deactivated everything is solved.
Thank you
